e2e/testcafe-devextreme/tests/navigation/splitter/events.ts

```diff
@@ -0,0 +1,25 @@
+import url from '../../../helpers/getPageUrl';
+import { createWidget } from '../../../helpers/createWidget';
+import Splitter from 'devextreme-testcafe-models/splitter';
+
+fixture.disablePageReloads`Splitter_events`
+  .page(url(__dirname, '../../container.html'));
+
+  test('Panes should not be able to resize when onResizeStart event canceled', async (t) => {
```

Comment:
Why we can't to write this test using qunit?

Reply:
have not managed to do it with pointerMock's drag dragStart etc, but probablty pointer.down().move, i will try again

Reply:
i tried and we decided to leave testcafe test